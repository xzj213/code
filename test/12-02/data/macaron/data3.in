4
120 3518 2913 1785
961 3230 2663 3817
1598 4094 3750 3193
2144 1929 516 1886
