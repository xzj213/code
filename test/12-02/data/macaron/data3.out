0 2 495 534 667 703 848 875 960 998 1087 1310 1425 1695 1721 1901 1927 1982 2242 2816 3095 3233 3369 3677 
